binaries/daemon/src/lib.rs

```diff
@@ -1543,17 +1544,19 @@ impl RunningDataflow {
                     let span = tracing::span!(tracing::Level::TRACE, "tick");
                     let _ = span.enter();
 
+                    let mut parameters = BTreeMap::new();
+                    parameters.insert(
+                        "open_telemetry_context".to_string(),
+                        #[cfg(feature = "telemetry")]
+                        Parameter::String(serialize_context(&span.context())),
+                        #[cfg(not(feature = "telemetry"))]
+                        Parameter::String("".into()),
```

Comment:
We could even omit the parameter instead of using an empty string.

Reply:
Kind of prefer to keep an empty string as this would be a breaking change and might error out a lot of legacy code.